kpi/views.py

```diff
@@ -85,13 +87,30 @@
 from .utils.ss_structure_to_mdtable import ss_structure_to_mdtable
 from .tasks import import_in_background
 from deployment_backends.backends import DEPLOYMENT_BACKENDS
+from ona.authentication import (
+    JWTAuthentication, encode_payload, decode_payload
+)
+
 
 CLONE_ARG_NAME = 'clone_from'
 COLLECTION_CLONE_FIELDS = {'name'}
 
 
-@login_required
 def home(request):
+    if request.user.is_anonymous():
+        cookie_jwt = request.COOKIES.get('__enketo')
+        if cookie_jwt:
+            auth_class = JWTAuthentication()
+            user, token = auth_class.authenticate(request)
+            user.backend = settings.AUTHENTICATION_BACKENDS[0]
+            login(request, user)
+
+            return TemplateResponse(request, "index.html")
+        else:
+            return HttpResponse(
+                ("Login to http://ona.io/login on a different browser tab"
```

Comment:
When does the user see a raw URL like this? Or does this never get to the user? If so we'll want to show them something prettier, or just redirect them.